### PR TITLE
Authentication request log with IP

### PR DESCRIPTION
**Pull request by dafo90**
**Changes**

Added IP to authentication request log
**Issues**

### CODE REVIEW

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 1]
```diff
@@ -357,7 +357,7 @@ public string MakeValidUsername(string username)
                 IncrementInvalidLoginAttemptCount(user);
             }
 
-            _logger.LogInformation("Authentication request for {0} {1}.", user.Name, success ? "has succeeded" : "has been denied");
+            _logger.LogInformation("Authentication request for {0} {1} (IP: {2}).", user.Name, success ? "has succeeded" : "has been denied", remoteEndPoint);
```
Comment (mark-monteiro) [inline Emby.Server.Implementations/Library/UserManager.cs:360]: The paramaters in the log template string should be specified like `{UserName}`, `{Success}`, etc (see [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation?view=dotnet-plat-ext-3.1#Microsoft_Extensions_Logging_LoggerExtensions_LogInformation_Microsoft_Extensions_Logging_ILogger_Microsoft_Extensions_Logging_EventId_System_Exception_System_String_System_Object___)).

**Comment by mark-monteiro** [comment]
I wonder if there are privacy concerns with logging IP addresses alongside usernames? Maybe it would be better to only log the IP address on failed attempts?

**Comment by dafo90** [comment]
Yesterday I fixed the log as you request and added some more

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 2]
```diff
@@ -319,6 +320,7 @@ public string MakeValidUsername(string username)
 
             if (user == null)
             {
+                _logger.LogInformation("Authentication request for {UserName} has been denied (IP: {IP}).", username, remoteEndPoint);
```
Comment (mark-monteiro) [inline Emby.Server.Implementations/Library/UserManager.cs:323]: Why log here but not for the other `AuthenticationException`s that could be thrown below?

**Pull request merged**